frontend/src/components/Mobile/InputField.tsx

```diff
@@ -1,24 +1,45 @@
 import React, { Dispatch, SetStateAction, useState } from 'react'
 import styled from 'styled-components'
+import { FieldValues, UseFormRegister } from 'react-hook-form'
 
 import { Input } from 'antd'
 
+type InputFieldProps = {
+  title: string
+  placeholder: string
+  value?: string
+  setValue?: Dispatch<SetStateAction<string>> | ((input: string) => void)
+  textArea?: boolean
+  onChange?: () => void
+  errors: { [x: string]: any }
+  isRequired?: boolean
+  register: UseFormRegister<FieldValues>
```

Comment:
should this be optional instead?

Reply:
[d3f894c](https://github.com/rhdevs/rhapp/pull/614/commits/d3f894cadaa0993923546ac8e1102cfa56d1ffac)